packages/devtools-contextmenu/package.json

```diff
@@ -9,6 +9,10 @@
   "author": "Jason Laster",
   "license": "MPL-2.0",
   "dependencies": {
-    "devtools-modules": "^0.0.38"
+    "autoprefixer": "^9.1.5",
+    "devtools-modules": "^0.0.38",
+    "postcss-bidirection": "^2.7.0",
+    "postcss-class-namespace": "^0.1.0",
+    "postcss-url-mapper": "^1.2.0"
```

Comment:
why do we need to add these css tools?

Reply:
@jasonLaster if I remove the postcss tools debugger fails to build. Can we let them in?